src/file_pipe_log.rs

```diff
@@ -35,12 +36,37 @@ const REWRITE_NAME_LEN: usize = REWRITE_NUM_LEN + REWRITE_SUFFIX_LEN;
 const INIT_FILE_NUM: u64 = 1;
 
 pub const FILE_MAGIC_HEADER: &[u8] = b"RAFT-LOG-FILE-HEADER-9986AB3E47F320B394C8E84916EB0ED5";
-pub const VERSION: &[u8] = b"v1.0.0";
 const DEFAULT_FILES_COUNT: usize = 32;
 #[cfg(target_os = "linux")]
 const FILE_ALLOCATE_SIZE: usize = 2 * 1024 * 1024;
 
-pub struct LogFd(RawFd);
+#[derive(Clone, Copy)]
+pub enum Version {
+    V1 = 1,
+}
+
+impl Version {
+    fn current() -> Self {
+        Self::V1
+    }
+
+    fn len() -> usize {
+        8
+    }
+
+    fn as_u64(&self) -> u64 {
+        *self as u64
+    }
+
+    fn from_u64(v: u64) -> Result<Self> {
+        match v {
+            1 => Ok(Self::V1),
+            _ => Err(Error::UnrecognizedLogFileVersion(v)),
+        }
+    }
+}
+
+pub struct LogFd(RawFd, Version);
```

Comment:
No need to hold `Version`, in the future `Version` can translate to a set of concrete flags that can be stored here.